test/start.test.js

```diff
@@ -1034,3 +1034,24 @@ test('should start fastify with custom plugin options with a CJS plugin with pac
   t.pass('server closed')
   t.end()
 })
+
+test('should throw error for invalid fastify plugin (object)', async t => {
+  t.plan(1)
```

Comment:
I count 2 assertion but only 1 is planned - so one of the two is not executed

Reply:
Should be updated now @Eomm